org.mwc.cmap.legacy/src/MWC/GUI/Shapes/Symbols/SVG/SVGElementFactory.java

```diff
@@ -0,0 +1,62 @@
+/*
+ *    Debrief - the Open Source Maritime Analysis Application
+ *    http://debrief.info
+ *
+ *    (C) 2000-2018, Deep Blue C Technology Ltd
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the Eclipse Public License v1.0
+ *    (http://www.eclipse.org/legal/epl-v10.html)
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. 
+ */
+package MWC.GUI.Shapes.Symbols.SVG;
+
+import org.jsoup.nodes.Element;
+
+public class SVGElementFactory
+{
+  private Element _svgElement;
+  
```

Comment:
In a _normal_ factory pattern implementation, we have a single factory instance, and that instance returns relevant new child instances from a `getXXX()` method:
https://www.tutorialspoint.com/design_pattern/factory_pattern.htm

So, I think this should have an empty constructor, and should not have the internal `_svgElement` member.  All it needs is for `getInstance()` to take a `svgElement` as a parameter.